libraries/WDT/examples/Example2_WDT_Config/Example2_WDT_Config.ino

```diff
@@ -0,0 +1,103 @@
+/*
+  Author: Adam Garbo
+  Created: August 1st, 2020
+  License: MIT. See SparkFun Arduino Apollo3 Project for more information
+
+  This example demonstrates how to modify the configuration of the Artemis
+  Watchdog Timer (WDT).
+
+  The watchdog timer is controlled by a clock divider, interrupt ticks and 
+  reset ticks. To achieve desired watchdog timing, a simple calculation can be made:
+
+  period = # ticks / clock divider frequency 
+
+  Examples: 
+  128 interrupt ticks / 128 Hz clock = 1 second
+  64 interrupt ticks / 16 Hz clock = 4 seconds
+  32 interrupt ticks / 1 Hz clock = 32 seconds
+  16 interrupt ticks / 1/16 Hz clock = 256 seconds
+  
+  The following code will configure the watchdog for both interrupt and reset
+  generation, and immediately start the watchdog timer.
+  The watchdog ISR provided will 'pet' the watchdog four times. On the fifth
+  interrupt, the watchdog will not be pet, so the 'reset' action can occur.
+  On the sixth timeout event, the WDT will issue a system reset, and the
+  program will start over from the beginning.
+
+  This example is based on the Ambiq SDK 2.4.2 watchdog.c example.
+
+  Tested with SparkFun Artemis Redboard.
+*/
+
+#include <WDT.h>
+
+APM3_WDT wdt;
+
+volatile bool watchdogFlag = false; // Watchdog Timer ISR flag
+volatile int watchdogInterrupt = 0; // Watchdog interrupt counter
+
+unsigned long currentMillis = 0,
+              previousMillis = 0;
+
+void setup()
+{
+  Serial.begin(115200);
+
+  Serial.println("Artemis Watchdog Timer Example");
+
+  // Configure the watchdog
+  /*
+    Available watchdog timer clock dividers:
+    0 = Low Power Mode. This setting disables the watch dog timer
+    1 = 128 Hz 
+    2 = 16 Hz 
+    3 = 1 Hz
+    4 = 1/16th Hz
+  */
+  // Set watchdog timer clock to 128 Hz
+  // Set watchdog interrupt to 1 seconds (128 ticks / 128 Hz = 1 second)
+  // Set watchdog reset ~2 seconds (255 ticks / 128 Hz = 1.99 seconds)
+  wdt.configure(1, 128, 255); // Note: Ticks are limited to 255 (8-bit)
```

Comment:
Use macro AM_HAL_WDT_LFRC_CLK_16HZ instead of 2?

Reply:
See: https://github.com/sparkfun/Arduino_Apollo3/pull/242#discussion_r465005471